lib/provider/camunda/parts/ListenerDetailProps.js

```diff
@@ -138,7 +138,7 @@ module.exports = function(group, element, bpmnFactory, options, translate) {
           listener = getSelectedListener(element, node),
           listenerType = ImplementationTypeHelper.getImplementationType(listener);
 
-      value.listenerValueLabel = LISTENER_TYPE_LABEL[listenerType] || '';
+      value.listenerValueLabel = translate(LISTENER_TYPE_LABEL[listenerType] || '');
```

Comment:
I'd rather not invoke `translate` with variables as it might be difficult to extract the template for localization.

Reply:
ok

Reply:
changed it.